src/interfaces/jira.ts

```diff
@@ -86,17 +86,29 @@ export interface JiraAssociation {
 }
 
 export interface JiraRemoteLinkData {
-	product: string;
 	remoteLinks: JiraRemoteLink[];
 }
 
+interface IssueKeysAssociation {
+	associationType: string;
+	values: string[];
+}
+
 export interface JiraRemoteLink {
 	id: string;
 	schemaVersion: string;
 	updateSequenceNumber: number;
 	associations: JiraAssociation[];
 	displayName: string;
+	description: string;
 	url: string;
 	type: string;
+	status: JiraRemoteLinkStatus;
 	lastUpdated: number;
+	associations: IssueKeysAssociation[];
```

Comment:
As the build says, you've got associations defined twice in this schema

Reply:
Resolved with merge conflict